docs/UserGuide.adoc

```diff
@@ -365,20 +371,260 @@ https://www.ura.gov.sg/realEstateIIWeb/resources/misc/list_of_postal_districts.h
 |79, 80 |Seletar
 |===
 
+
+[start=2]
+. By *area*: +
+Command Usage: `nearby` `AREA`
+
+[NOTE]
+====
+An *area* refers to one of the five areas of Singapore +
+
+. Central
+. East
+. North East
+. West
+. North
+
+You can obtain more information about each area from
+https://keylocation.sg/singapore/districts-map[here] +
+====
+* You can view the orders located at a specified *area*.
+
+
+More information about the 5 areas of Singapore are shown below: +
+
+.Central
+|===
+|District |Areas |Location
+
```

Comment:
I feel that this whole chunk can put in appendix and provide a link to it, at the same time at the appendix provide a link back to this particular section.

Something like
```
====
* You can view the orders located at a specified *area*.


More information about the 5 areas of Singapore are shown in <<appendix_region, Appendix region>>
```

...

```
[[appenix_region]]
the whole table here.
```